packages/plugins/typescript/resolvers/tests/mapping.spec.ts

```diff
@@ -920,12 +998,12 @@ describe('ResolversTypes', () => {
       ChildUnion: ResolverTypeWrapper<any>;
       Query: ResolverTypeWrapper<{}>;
       Subscription: ResolverTypeWrapper<{}>;
-      Node: ResolversTypes['SomeNode'];
+      Node: ResolverTypeWrapper<any>;
       ID: ResolverTypeWrapper<any>;
       SomeNode: ResolverTypeWrapper<any>;
-      AnotherNode: ResolversTypes['AnotherNodeWithChild'] | ResolversTypes['AnotherNodeWithAll'];
-      WithChild: ResolversTypes['AnotherNodeWithChild'] | ResolversTypes['AnotherNodeWithAll'];
-      WithChildren: ResolversTypes['AnotherNodeWithAll'];
+      AnotherNode: ResolverTypeWrapper<any>;
+      WithChild: ResolverTypeWrapper<any>;
+      WithChildren: ResolverTypeWrapper<any>;
```

Comment:
In the case of default mapper without placeholder, we can treat interface implementing types the same as how union treats it: typing it as `ResolverTypeWrapper<DefaultMapper>` instead of `ResolversTypes['typeA'] | ResolversTypes['typeB']`

The reason being `ResolversTypes['typeA'] | ResolversTypes['typeB']` resolves to `ResolverTypeWrapper<DefaultMapper> | ResolverTypeWrapper<DefaultMapper>` so we can simplify it.